errors.go

```diff
@@ -70,3 +75,10 @@ func (me *MySQLError) Is(err error) bool {
 	}
 	return false
 }
+
+func (me *MySQLError) IsWithSQLState(err error) bool {
+	if merr, ok := err.(*MySQLError); ok {
+		return merr.Number == me.Number && merr.SQLState == me.SQLState
+	}
+	return false
+}
```

Comment:
Uh, please wait.

`Is()` is designed to work with `errors.Is()`.

Obviously, `errors.Is()` don't use `IsWithSQLState()`. So same design don't work.
User need to use `errors.As()`.

Please remove this method for now.

Reply:
Removed